README.md

```diff
@@ -10,6 +10,7 @@ Note : 2019 conferences are listed [here](year-pages/2019-conferences-list.md).
 
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
+|[SciPy India](https://scipy.in)|21-22 Dec | LCH,IIT Bombay | International conference on Python for education and scientific computing. |
```

Comment:
Can you remove the `LCH` part from the venue? Just `IIT Bombay` should be enough :)

nit: also, can you add a space between the `|` and the opening bracket `[` i.e. `| [SciPy India]` and `| 21- 22 Dec |`